addon/components/infinity-loader.js

```diff
@@ -49,19 +63,19 @@ export default Ember.Component.extend({
     }
   },
 
-  _setupScrollable() {
-    var scrollable = this.get('scrollable');
-    if (Ember.typeOf(scrollable) === 'string') {
-      var items = Ember.$(scrollable);
+  _setupEvent(type) {
```

Comment:
nitpick - but technically we're searching / setting up the bindable dom element for the event, rather than the event itself. 

Maybe we should call this `_setupBindable` or something instead?